test/_common.py

```diff
@@ -0,0 +1,9 @@
+import platform
+
+replacements = dict([(platform.node(), "{HOSTNAME}")])
+
+
+def normalize_output(str):
+    for replacement in replacements:
+        str = str.replace(replacement, replacements[replacement])
+    return str
```

Comment:
`str` is a Python type so we should not shadow it.
```suggestion
def normalize_output(s):
    for replacement in replacements:
        s = s.replace(replacement, replacements[replacement])
    return s
```